enslig-forsorger/src/main/kotlin/no/nav/familie/kontrakter/ef/søknad/Dokumentasjon.kt

```diff
@@ -0,0 +1,3 @@
+package no.nav.familie.kontrakter.ef.søknad
+
+data class Dokumentasjon(val harSendtInnTidligere: Boolean, val dokumenter: List<Dokument>)
```

Comment:
Lurer nesten på om harSendtInnTidligere bør wrappes inn i et Søknadsfelt slik pdf-genereringen blir mer riktig
@bent-lorentzen @ma10s @charliemidtlyng

Reply:
evt. må vi ha spesialhåndtering for dokumentasjon uansett for pdf-generering? eller? Får jo ikke ut om du har sendt inn noe i lista heller ?

Reply:
Det er riktig, då blir heller ikke noe generert
Alternativet nå, med denne løsningen er att man tar med en tekst i pdf-genereringen https://github.com/navikt/familie-ef-mottak/pull/164/files#diff-bb1a36c579fdeafba32072b94624d925R30
Men syns det blir feil måte å løse det på